### PR TITLE
Day-1-and-2-cleanup.txt

### PR DESCRIPTION
**Pull request by Parzivus**
I'm going to finish the last two scenes in day 2 and then get the ones I skipped on day 1. Once that's done, I think every scene in Arc's route through day 5 will have been proofread by at least three people, so that's cool.

### CODE REVIEW

**Comment by Parzivus** [comment]
Gonna grab some Ciel scenes that show up as well. Done by Christmas?

**Comment by Parzivus** [comment]
Just realized I already proofread the last one and still found more stuff to change. Go figure.

**Comment by Parzivus** [comment]
Pretty sure that last one has a lot of overlap with the other choices - I'll check it in the morning.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt [hunk 1]
```diff
@@ -30,13 +30,13 @@ All right. In that case, come by the tea ceremony club after school. You're more
...
 <Page11>
-"It's a promise then. Between you and me, we have a surplus of snacks purchased with club funds. It's not something that stays fresh long, so I need a healthy boy to help out."
+"It's a promise, then. Between you and me, we have a surplus of snacks purchased with club funds. It's not the kind of thing that stays fresh long, so I need a healthy boy to help out." //Same edit as the previous file, keep both or neither.
 "I-Is that so? Well, if my schedule permits, I'll certainly take you up on that offer.
 Although, getting taken in by food is something a cat would do."
 <Page12>
 Her smile is so friendly that I can't help but joke a little.
 <Page13>
-The Senpai with a green ribbon replies,
+The Senpai with a green ribbon replies, //Maybe use this for the "ribbon-wearing" bit from the last file? I think a quick check between these two scenes would be good, this one is essentially the other one minus Shiki being horny.
```
Comment (Gekimon) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt:39]: I'm personally not a fan of using "a/the + senpai/sensei etc."
I would replace with `Senpai, who was wearing a green ribbon, replies,`
Or `The upperclassman with a green ribbon replies`

And if we're keeping "the senpai", then maybe without a capital S?

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt [hunk 2]
```diff
@@ -25,7 +25,7 @@ For some reason, I imagine a funeral with no attendees or a black and white stri
 I decide to shake off my unpleasant premonition and head to the classroom.
 <Page7>
 I enter the empty classroom.
-I sit down on my chair and kill time until the morning homeroom.
+I sit down in my chair and kill time until the morning homeroom.
```
Comment (Gekimon) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt:28]: Beta_testers_suggestions_2021_12_05 already changed that once, from "in" to "on".
Is it "sit down on my chair" or "sit down in my chair"?

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 3]
```diff
@@ -13,22 +13,22 @@ Or is she coming straight towards me...?
 "Oh good, I've been looking for you, Tohno-kun.
 You saved me the trouble of having to visit your classroom."
 <Page6>
-"W-Well, that's great. Hello, Ciel-senpai."
+"W-Well, that's great. Good morning, Ciel-senpai."
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:16]: I changed this one because Ciel mentions how you wouldn't say こんにちは the second time you meet someone in the same day. I don't think "hello" really has the same connotation. Any better terms to use here?

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt (hunk 1 above)
Comment (Parzivus) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt:39]: "Upperclassman" flows pretty well here, I'll put that in for now.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt [hunk 4]
```diff
@@ -52,7 +52,7 @@ Vicious words reverberate in the classroom as if to ignore the fact that classma
 I don't need to see it to confirm it. The atmosphere in the classroom has changed because of the arrival of this―――
 <Page14>
 "Sup! Good mornin' four eyes!
-Man you look irritated this morning. You seriously look like shit!"
+Man, you look irritated this morning. You seriously look like shit!" //I think a couple of these were caught by beta testers recently, I'll add them in as I find them.
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt:55]: You're not the only one to mention this, I think I'll revert it.

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 5]
```diff
@@ -85,7 +85,7 @@ What is that?!
 What is that?!
 <Page21>
 A scream as if it had lost its mind.
-Or like a gale.
+Or a powerful gale. //Starting a sentence with "Or" feels weird, even though it's accurate, but lengthing it doesn't feel right either.
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:88]: Maybe "howling like a gale"?

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 6]
```diff
@@ -76,7 +76,7 @@ Like nothing I've ever seen before (it's not something you'd normally see)
 An impossible sight―――
 <Page20>
                                     A melting dog.
-What is that?!
+What is that?! //Even for なんだアレは, the exclamation point seems a little strong. Thoughts?
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:79]: Looking back at it, I don't have a super strong preference for what punctuation to use here. Anyone else have an opinion? Might just leave as is otherwise.

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 7]
```diff
@@ -94,7 +94,7 @@ It fades away without a trace.
 Is what I'm seeing now a hallucination or a dream?
 <Page23>
 ―――Then, is that an illusion too?
-There, under the streetlight that slices through the darkness, stands an obscure figure.
+There, under the streetlight that slices through the darkness, stands an obscured figure. //Sounds too much like the person is obscure, as opposed to their appearance.
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:97]: "There stands a figure I can't comprehend" works pretty well, I think.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt [hunk 8]
```diff
@@ -67,6 +68,25 @@ The Tohno family was a wealthy family and also shareholders of several companies
...
+Akiha must have hated me... Tohno Shiki.
+Or rather, she has every right to hate me.
+She was left alone in that huge mansion, living with my hard-headed father.
+It's easy to imagine how Akiha feels about me running away from her.
+<Page18>
+The Tohno family was a wealthy family and also shareholders of several companies. My father, Tohno Makihisa, was apparently a mediocre businessman, but the assets he inherited from his grandfather were strong enough to keep the company running smoothly.
+<Page19>
+An anachronistic building.
+The life of the upper class, respecting civility and history.
+Such a life in the mansion seemed boring to my childish mind.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt:89]: uhhh what's goin on here

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt [hunk 9]
```diff
@@ -58,6 +58,7 @@ It's that, you know. I just wanted to make sure you weren't feeling blue."
 But this is one step short of horror! It's so cruel I can't even laugh!"
 'Ah ha ha', laughs Arihiko exaggeratedly. One of his strong points is that he doesn't take himself too seriously no matter what.
 <Page17>
+<Page17>
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt:61]: Duplicate line

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt [hunk 10]
```diff
@@ -30,13 +30,13 @@ All right. In that case, come by the tea ceremony club after school. You're more
...
 <Page11>
-"It's a promise then. Between you and me, we have a surplus of snacks purchased with club funds. It's not something that stays fresh long, so I need a healthy boy to help out."
+"It's a promise, then. Between you and me, we have a surplus of snacks purchased with club funds. It's not the kind of thing that stays fresh long, so I need a healthy boy to help out." //Same edit as the previous file, keep both or neither.
 "I-Is that so? Well, if my schedule permits, I'll certainly take you up on that offer.
 Although, getting taken in by food is something a cat would do."
 <Page12>
 Her smile is so friendly that I can't help but joke a little.
 <Page13>
-The Senpai with a green ribbon replies,
+The upperclassman with a green ribbon replies, //Maybe use this for the "ribbon-wearing" bit from the last file? I think a quick check between these two scenes would be good, this one is essentially the other one minus Shiki being horny.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt:39]: Update the other file with this too

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:88]: As the full line? Doesn't feel quite right. I honestly think the original line is fine here; gale already encodes the strength of the winds. @Avuxo @yk1512 thoughts?

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:16]: Hmm, "Good to see you"? A bit wordy. Morning just seems off since, well, it's lunchtime now so it's probably after noon.

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:88]: I think the original line is fine here.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (yk1512) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:16]: I'm fine with 'good to see you'

**Review by yk1512 — APPROVED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt (hunk 8 above)
Comment (Parzivus) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt]: I'm a dummy and copy-pasted wrong when trying to make it match the other file. It's fixed now.
